### PR TITLE
feat(w3c/groups): add support for CGs

### PR DESCRIPTION
**Pull request by sidvishnoi**
Closes https://github.com/marcoscaceres/respec.org/issues/100

### CODE REVIEW

File: routes/w3c/groups.json [hunk 1]
```diff
@@ -1,36 +1,370 @@
...
+    "webrtc": 47318,
+    "webxr": 109735,
+    "wot": 95969
+  },
+  "cg": {
+    "abcg": 118199,
+    "accessibility4children": 119612,
+    "accessibilityinindia": 61068,
+    "accesslearn": 77612,
+    "act-r": 114156,
```
Comment (sidvishnoi) [inline routes/w3c/groups.json:43]: We probably don't want all of these as not all are active.

**Comment by sidvishnoi** [comment]
image

File: routes/w3c/groups.json (hunk 1 above)
Comment (marcoscaceres) [reply routes/w3c/groups.json:43]: can you filter them?

Comment (sidvishnoi) [reply routes/w3c/groups.json:43]: Maybe sort them by participant count?

Comment (marcoscaceres) [reply routes/w3c/groups.json:43]: let's leave them for now... alt, we ask the w3c for better data here.

Comment (sidvishnoi) [reply routes/w3c/groups.json:43]: Also, "wot" is shortname for both a WG and a CG

**Pull request merged**